WebCryptoAPI/sign_verify/eddsa_vectors.js

```diff
@@ -34,25 +36,48 @@ function getTestVectors() {
   var signatures = {
       "Ed25519": new Uint8Array([61, 144, 222, 94, 87, 67, 223, 194, 130, 37, 191, 173, 179, 65, 177, 22, 203, 248, 163, 241, 206, 237, 191, 74, 220, 53, 14, 245, 211, 71, 24, 67, 164, 24, 97, 77, 203, 110, 97, 72, 98, 97, 76, 247, 175, 20, 150, 249, 52, 11, 60, 132, 78, 164, 220, 234, 177, 211, 209, 85, 235, 126, 204, 0]),
       "Ed448": new Uint8Array([118, 137, 126, 140, 80, 172, 107, 17, 50, 115, 92, 9, 197, 95, 80, 108, 1, 73, 210, 103, 124, 117, 102, 79, 139, 193, 11, 130, 111, 189, 157, 240, 160, 60, 217, 134, 188, 232, 51, 158, 100, 199, 209, 114, 14, 169, 54, 23, 132, 220, 115, 131, 119, 101, 172, 41, 128, 192, 218, 192, 129, 74, 139, 193, 135, 209, 201, 201, 7, 197, 220, 192, 121, 86, 248, 91, 112, 147, 15, 228, 45, 231, 100, 23, 114, 23, 203, 45, 82, 186, 183, 193, 222, 190, 12, 168, 156, 206, 203, 205, 99, 247, 2, 90, 42, 90, 87, 43, 157, 35, 176, 100, 47, 0]),
-  }
+      "Ed25519-raw": new Uint8Array([197, 150, 209, 125, 190, 91, 214, 228, 254, 248, 133, 80, 75, 235, 17, 40, 176, 54, 107, 94, 32, 56, 182, 227, 165, 230, 0, 135, 96, 165, 235, 169, 141, 11, 95, 171, 169, 66, 236, 161, 31, 76, 39, 92, 128, 229, 90, 66, 171, 212, 209, 108, 157, 116, 82, 61, 90, 149, 230, 245, 191, 10, 92, 3])
```

Comment:
Why are we adding "Ed25519-raw", it does not seem like we are using signatures["Ed25519-raw"] anywhere.

Reply:
This does not follow https://wicg.github.io/webcrypto-secure-curves/, there is no new `Ed25519-raw` being added.